packages/inngest/jest.config.js

```diff
@@ -6,7 +6,7 @@ module.exports = {
   roots: ["<rootDir>/src"],
   moduleNameMapper: {
     "(\\..+)\\.js": "$1",
-    inngest: "<rootDir>/src",
+    "^inngest$": "<rootDir>/src",
```

Comment:
This regex was picking up anything containing `inngest` and redirecting it to `src/` - bit of a problem when importing `@inngest/test`!